prusti-viper/src/encoder/mir/types/encoder.rs

```diff
@@ -27,6 +27,12 @@ pub struct TypeEncoder<'p, 'v: 'p, 'tcx: 'v> {
     ty: ty::Ty<'tcx>,
 }
 
+// TODO: move this?
+fn encode_lifetime_name(lft: String) -> String {
+    // TODO: check if all lifetime names have the parttern '_#7r
```

Comment:
That number can be more that one digit long...

Reply:
Yes, currently the first 3 and the last digits are removed and "lft" is prefixed